specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/examples/Certificate_CreateOrUpdate.json

```diff
@@ -8,8 +8,8 @@
     "certificateEnvelope": {
       "location": "East US",
       "properties": {
-        "password": "private key password",
-        "value": "Y2VydA==",
+        "password": "<Password>",
+        "value": "<value>",
```

Comment:
Same change is need for:
![image](https://github.com/Azure/azure-rest-api-specs/assets/17716005/e3b5958b-e69e-4e82-bf30-7d6dbd36b47f)